Makefile

```diff
@@ -49,8 +48,9 @@ ifndef NOCOLOR
 	RESET  := $(shell tput -Txterm sgr0)
 endif
 
+BUILD_TOOLS_TARGET_DIR ?= /build
 DOCKER_RUN_FLAGS += --rm
-DOCKER_RUN_FLAGS += -v ${PWD}:/app/src
+DOCKER_RUN_FLAGS += -v ${PWD}:${BUILD_TOOLS_TARGET_DIR}
```

Comment:
this is not dynamic but fixed... so the variable is not needed... we can never change it...

Reply:
and in the end why is it build and not src anymore? arn't we mounting sources here?

Reply:
in addition this is used in all modules... so changing this will need to adjust everythere... what is the reason to not use /app/src ?

Reply:
yes I am gonna change this to `/build` in all directories - `/app/src` doesn't make sense, it's a cosmetic change. Since I am touching all repos anyways it doesn't hurt

Reply:
resoilved the variable in 5dcff2f